src/main/java/Lotto.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    private final ArrayList<Integer> nums;
+
+    public int numCount() {
+        return this.nums.size();
+    }
+
+    public Lotto(ArrayList<Integer> nums){
+        if(nums.stream().anyMatch(num -> num < 1 || num > 46)){
+            throw new IllegalArgumentException("로또 번호는 1~46 범위이어야 합니다.");
+        }
+
+        this.nums = nums;
+    }
+
+    public Lotto(String numStr){
+        this.nums = Arrays.stream(numStr.split(","))
+                .map(Integer::parseInt)
+                .collect(Collectors.toCollection(ArrayList::new));
+    }
```

Comment:
```suggestion
    public Lotto(String numStr){
        this(Arrays.stream(numStr.split(","))
                .map(Integer::parseInt)
                .collect(Collectors.toCollection(ArrayList::new)));
    }
```
부 생성자는 주 생성자를 호출해 구현하는 것을 추천